README.md

```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
-* [Stark.net](https://starknet.io/) - StarkWare is building StarkNet, a decentralized, permissionless and censorship-resistant STARK-powered L2 ZK-Rollup that supports general-computation over Ethereum. It is based on the Turing-complete Cairo language.
-* [Eigen Layer](https://www.eigenlayer.com/) - Eigen layer is a general-purpose marketplace for decentralized trust, which is built on Ethereum, the largest programmable decentralized trust network and unbundles the trust layer of Ethereum so components of the trust network can be reused for other purposes.
+* [Stark.net](https://starknet.io/) - StarkWare is building StarkNet, a decentralized, permissionless, and censorship-resistant STARK-powered L2 ZK-Rollup that supports general computation over Ethereum. It is based on the Turing-complete Cairo language.
+* [Eigen Layer](https://www.eigenlayer.com/) - Eigen layer is a general-purpose marketplace for decentralized trust, which is built on Ethereum, the largest programmable decentralized trust network, and unbundles the trust layer of Ethereum so components of the trust network can be reused for other purposes.
 * [Fuel Network](https://www.fuel.network/) - Fuel v1 began as a layer-2 (L2) scalability technology for a monolithic Ethereum. It was the first optimistic rollup on mainnet Ethereum, deployed at the end of 2020.
 * [Scroll.io](https://scroll.io/) - Scroll is a zkEVM-based zkRollup on Ethereum that enables native compatibility for existing Ethereum applications and tools.
 * [Celestia](https://celestia.org) - Celestia is the first modular blockchain network that makes it effortless to deploy a new blockchain.
-* [ZKSpace](https://zkspace.info/) - The ZKSpace platform consists of three main parts: a Layer 2 AMM DEX utilizing ZK-Rollups technology ZKSwap v3, a payment service called ZKSquare, and an NFT marketplace called ZKSea.
-* [Aztec Connect](https://aztec.network/) - Aztec Connect is an open source layer 2 network that aims to bring scalability and privacy to Ethereum.
+* [ZKSpace](https://zkspace.info/) - The ZKSpace platform consists of three main parts: a Layer 2 AMM DEX utilizing ZK-Rollups technology ZKSwap v3, a payment service called ZKSquare, an NFT marketplace called ZKSea.
+* [Aztec Connect](https://aztec.network/) - Aztec Connect is an open-source layer 2 network that aims to bring scalability and privacy to Ethereum.
 * [zkSync](https://zksync.io/) - zkSync is a scaling and privacy engine for Ethereum.
-* [Golem](https://www.golem.network/) - Golem is a global, open-source, decentralized supercomputer that anyone can use. It is made up of the combined computing power of the users' machines, from PCs to entire data centers.
+* [Golem](https://www.golem.network/) - Golem is a global, open-source, decentralized supercomputer that anyone can use. It is made up of the combined computing power of the user's machines, from PCs to entire data centers.
```

Comment:
`users'` - it was correct first time

Reply:
How is Golem ZK?

Reply:
https://handbook.golem.network/payments/layer-2-payments 
From my point of understanding, they adopted the scalability of Layer 2 solutions for zk-projects such as zkSync or Polygon.  But maybe I am wrong about this.

Reply:
Exactly. It is using zk platforms, but is not zk itself. Please remove it.